spec/requests/api/v1/streaming_spec.rb

```diff
@@ -28,20 +27,33 @@
       Rails.configuration.x.streaming_api_base_url = "wss://streaming-#{Rails.configuration.x.web_domain}"
     end
 
-    describe 'GET #index' do
+    describe 'GET /api/v1/streaming' do
       it 'redirects to streaming host' do
-        get :index, params: { access_token: 'deadbeef', stream: 'public' }
-        expect(response).to have_http_status(301)
-        request_uri = URI.parse(request.url)
-        redirect_to_uri = URI.parse(response.location)
-        [:scheme, :path, :query, :fragment].each do |part|
-          expect(redirect_to_uri.send(part)).to eq(request_uri.send(part)), "redirect target #{part}"
-        end
-        expect(redirect_to_uri.host).to eq(streaming_host), 'redirect target host'
+        get '/api/v1/streaming', headers: headers, params: { access_token: 'deadbeef', stream: 'public' }
+
+        expect(response)
+          .to have_http_status(301)
+
+        expect(redirect_to_uri)
+          .to have_attributes(
```

Comment:
Chose to replace the previous `send` approach with this check instead.

Reply:
Looks good to me!